cms/static/js/views/active_video_upload_list.js

```diff
@@ -10,6 +11,7 @@ define(
                 "dragleave .file-drop-area": "dragleave",
                 "drop .file-drop-area": "dragleave"
             },
+            pageUnloadMessage: pageUnloadMessage,
```

Comment:
Is there a particular reason why you set the local `pageUnloadMessage` variable on line 5, and then only reference it here? Why not put the actual content on this line, and refer to `this.paveUnloadMessage` in the `onBeforeUnload` function?